kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/AbstractST.java

```diff
@@ -35,7 +35,7 @@
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @ExtendWith(TestVisualSeparatorExtension.class)
 @ExtendWith(KroxyliciousExtension.class)
-class AbstractST {
+public class AbstractST {
```

Comment:
Its quite unusual to see a JUnit5 class as public, was there a particular reason for the change?

Reply:
New filter test suites are in a different package location so it requires to be accessible from there.